types/selenium-webdriver/bidi/logInspector.d.ts

```diff
@@ -0,0 +1,29 @@
+import { FilterBy } from './filterBy';
+import { ConsoleLogEntry, JavascriptLogEntry, GenericLogEntry } from './logEntries';
+
+export const LOG: {
+    TYPE_CONSOLE: string;
+    TYPE_JS_LOGS: string;
+};
```

Comment:
I don't think this is exported in the [real implementation](https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/bidi/logInspector.js#L21).

Reply:
@SupernaviX - Thanks for the call out, removed from the d.ts file, thanks!